JupyterDocs/ariaMisclosure/ariaMisclosure_tutorial.ipynb

```diff
@@ -1,10 +1,5 @@
 {
```

Comment:
<p>ITs not clear what API key for what.</p><ul><li>Explain first also why you need a key for what you need it and then how to get it.</li></ul><p> </p><p> </p><p>Also you just need it period, so does not add much information.</p><ul><li>Remove on same computer as running this notebook as cloud is not a computer.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aria-tools/ARIA-tools-docs/pull/46/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>